tests/test_images.py

```diff
@@ -0,0 +1,87 @@
+import datetime
+import os
+import shutil
+import tempfile
+import unittest
+
+import pytz
+
+from server import images
+
+
+class ImagesTest(unittest.TestCase):
+
+    def setUp(self):
+        self.temp_dir = tempfile.mkdtemp()
+        self.indexer = images.Indexer(self.temp_dir)
+
+    def tearDown(self):
+        shutil.rmtree(self.temp_dir)
+
+    def test_indexes_image_files(self):
+        open(os.path.join(self.temp_dir, '2017-04-01T1851Z.jpg'), 'a').close()
```

Comment:
How is this working if you call ```close``` before ```indexer.index```[?!?!](http://www.reactiongifs.com/wp-content/uploads/2013/10/tim-and-eric-mind-blown.gif)